### ISSUE TITLE
Sometimes after a clean, software is build in root project

### ISSUE THREAD

**Issue by dradoudine** [15knots/cmake4eclipse#204]
If you have any question, please **consult the <a href="http://groups.google.com/d/forum/cmake4eclipse-users">mailing list</a> first** before you raise issues here.

**Describe the bug**
![image](https://github.com/15knots/cmake4eclipse/assets/173178454/bce263de-4b16-4894-bb3b-3b4af5db77ff)
WHen I don't set an output folder, il *create a _build folder where build is made.
Sometimes (it's aleatory), build is made on root project and I have to clean manually
**To Reproduce**
1. Clean
2. Build (this is quiet aleatory but after 1 day of work I see it often

**Expected behavior**
build file shall be in _build folder

**Screenshots**

**Useful Information**
- cmake4eclipse version: 5.0
- CDT version: 11.6
- Which OS do you use: windows
- Cmake version: 3.30

***What is in Eclipse's Error Log?***

Sanitized stack traces:

**Additional context**

**Comment by dradoudine** [comment]
it's the target clean of my cmake file

**Comment by dradoudine** [comment]
it seems this problems occurs when Build output location field is set to empty (by default it create _build folder, so in case of empty field, it should be set by default to a value)

**Comment by 15knots** [comment]
@dradoudine I agree, creating a default `_build` folder when the output location field is empty is confusing to users. This has been fixed now.
